Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:16
+
+RUN set -x \
+  && git clone https://github.com/DataBiosphere/terra-ui.git \
+  && cd terra-ui \
+  && git checkout dev \
```

Comment:
Should this image be built from the local copy of terra-ui instead of downloading from GitHub?

Would it not defeat the purpose of a BEE if the terra-ui service is always built from the tip of the `dev` branch?

Reply:
Good catch, thanks for this. Fixed!